build-setup-nolog.sh

```diff
@@ -136,10 +136,9 @@ if [ "$IS_LIBRARY" = true ]; then
     fi
 else
     # note: lock file must end in .conda-lock.yml - see https://github.com/conda-incubator/conda-lock/issues/154
-    LOCKFILE="$RDIR/conda-reqs/conda-reqs.conda-lock.yml"
```

Comment:
You still need to point to the lock file to use in the `conda-lock install` step below.

Reply:
Maybe you can make the path it dumps to a argument to the script?

Reply:
I'd prefer it if there was a single source of truth for the path to the lockfile, and for that to live in the script generating the lockfile (so it can be called without any args). Maybe build-setup can read the script and get the path that way?

Reply:
Hmm. What about this... the script has the hardcoded path (something like `RDIR/conda-reqs/lock/lockfile.conda-lock.yml`. Then in the `build-setup-nolog.sh` script does a `find` to get the name of the file looking in that exact location (`RDIR/conda-reqs/lock`). Or something like that (maybe you don't need the extra directory and instead just search in `conda-reqs` for any file that ends with `conda-lock.yml`).